front-end-master/about_us/about.js

```diff
@@ -1,8 +1,8 @@
-class Division{
-	 constructor{
-
-	 }
-}
+//class Division{
+//	 constructor{
+//
+//	 }
+//}
 
 class Person{
```

Comment:
Sorry that you have to do this way. React would have been so much nicer. We will get onto it soon. Not for this project.

Reply:
yes :'( react was much better